### PR TITLE
Added initial Locust tests.

### PR DESCRIPTION
**Pull request by pm7h**

### CODE REVIEW

**Comment by jkowalski** [comment]
Can you rebase and squish all commits into one?

**Comment by aLekSer** [comment]
In this PR `build/Makefile` was deleted by mistake.

File: build/index.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
...+d+f+"&zx="+google.time()+c;/^http:/i.test(a)&&google.https()&&(google.ml(Error("a"),!1,{src:a,glmm:1}),a="");return a};}).call(this);(function(){google.y={};google.x=function(a,b){if(a)var c=a.id;else{do c=Math.random();while(google.y[c])}google.y[c]=[a,b];return!1};google.lm=[];google.plm=function(a){google.lm.push.apply(google.lm,a)};google.lq=[];google.load=function(a,b,c){google.lq.push([[a],b,c])};google.loadAll=function(a,b){google.lq.push([a,b])};}).call(this);google.f={};</script><script nonce="cV6G+uNGqI2mF3HWdzPVOg==">var a=window.location,b=a.href.indexOf("#");if(0<=b){var c=a.href.substring(b+1);/(^|&)q=/.test(c)&&-1==c.indexOf("#")&&a.replace("/search?"+c.replace(/(^|&)fp=[^&]*/g,"")+"&cad=h")};</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}@media all{.gb1{height:22px;margin-right:.5em;vertical-align:top}#gbar{float:left}}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb4{color:#00c !important}.gbi .gb4{color:#dd8e27 !important}.gbf .gb4{color:#900 !important}
```
Comment (markmandel) [inline build/index.html]: I've got to ask - why do we have... what looks like a copy of the Google homepage in here? :confused: :smile:

Comment (pm7h) [reply build/index.html]: Honestly I have no memory of explicitly adding this! It must have been some git mistake on my side. Removed.

**Comment by markmandel** [comment]
Can we cleanup the commit history on this? Not sure what's going on :smiley: 

But then happy to take it for a spin - keen to put some numbers against some of the allocation refactoring that has been going on :+1:

**Comment by markmandel** [comment]
Is this on hold until we come to a solid conclusion on #573 ?

**Comment by pm7h** [comment]
> Is this on hold until we come to a solid conclusion on #573 ?

I think so. My impression from discussions and also #573 was that Fortio might replace Locust. Mainly because it has better graphing capabilities which allow non-time-series representation. Unless we think we will need Locust as an alternative?

**Comment by markmandel** [comment]
It doesn't sound like we will - but I'll defer to you and @jkowalski 's research on this & the design in #573  - you both seem to have a good handle on what you want to there.

**Pull request closed without merging**